src/xinterpreter.cpp

```diff
@@ -93,12 +96,14 @@ int __get_cxx_version () {
   }
 __get_cxx_version ()
       )";
-
-        auto cxx_version = Cpp::Evaluate(code);
-        return std::to_string(cxx_version);
+      auto cxx_version = Cpp::Evaluate(code);
+      return std::to_string(cxx_version);
+#else
+      constexpr int cxx_version = 20;
+      return std::to_string(cxx_version);
+#endif
```

Comment:
Repeating again, we are only sticking to one version of the kernel currently for xeus-cpp-lite.